Stock.h

```diff
@@ -0,0 +1,49 @@
+#pragma once
+#include <string>
+#include <map>
+
+class Stock {
+    public:
+        // Constructor
+        Stock(
+            std::string ticker, 
+            std::string companyName,
+            int currentPrice,
+            int shares,
+            int marketCap,
```

Comment:
Since these things are dynamic with time, I'm hesitant to include them inside of this class; however, we can leave them in for now given they provide some value.